Problem: docker compose build fails after git pull

it fails at the pytest stage, with the following results

35.90 =========================== short test summary info ============================
35.90 FAILED tests/test_eip712.py::TestEip712::test_domain_contract - AttributeErro...
35.90 FAILED tests/test_eip712.py::TestEip712::test_domain_mismatch - AttributeErro...
35.90 FAILED tests/test_eip712.py::TestEip712::test_preprocess - AttributeError: 'S...
35.90 FAILED tests/test_eip712.py::TestEip712::test_recover_message - AttributeErro...
35.90 FAILED tests/test_eip712.py::TestEip712::test_signature - AttributeError: 'Si...
35.90 FAILED tests/test_signer.py::TestSigner::test_eip712_signer - AttributeError:...
35.90 ============= 6 failed, 29 passed, 4 skipped, 6 warnings in 2.61s ==============

--- a/Dockerfile
+++ b/Dockerfile
@@ -7,4 +7,5 @@ RUN ["apt-get", "install", "-y", "git"]
 RUN mkdir /ingestor
 ADD . /ingestor
 WORKDIR /ingestor
-RUN pip install . && python -mpytest .
+RUN pip install . 
+RUN python -mpytest .

--- a/src/truflation/data/signer/__init__.py
+++ b/src/truflation/data/signer/__init__.py
@@ -157,7 +157,7 @@ class Eip712Signer(Signer):
         ic(sm)
         return {
             'sig': {
-                'hash': hexbytes_no0x_str(sm.messageHash),
+                'hash': hexbytes_no0x_str(sm.message_hash),
                 'signature': hexbytes_no0x_str(sm.signature)
             }
         }

